bluetooth/src/commonMain/kotlin/Bluetooth.kt

```diff
@@ -470,18 +469,11 @@ suspend fun Flow<Device?>.updateRssi() {
 
 /**
  * Attempts to request a [MTU] size for the [Device] from a [Flow] of [Device]
- * When this method completes, the devices should have had [ConnectableDeviceState.Connected.requestMtu] called
  * @param mtu the [MTU] size to request
- * @return if `true` the new MTU value has been requested successfully
  */
-suspend fun Flow<Device?>.requestMtu(mtu: MTU): Boolean = state().transformLatest { deviceState ->
-    when (deviceState) {
-        is ConnectableDeviceState.Connected -> {
-            emit(deviceState.requestMtu(mtu))
-        }
-        else -> {}
-    }
-}.first()
+suspend fun Flow<Device?>.requestMtu(mtu: MTU) = state()
```

Comment:
Ideally you'd probably want to return the DeviceAction here so its completedSuccessfully can be awaited